binary_search_tree/binary_search_tree.py

```diff
@@ -10,13 +10,41 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    if self.value:
```

Comment:
you _might_ be able to skip this since BST is initialized with value. good thinking about edge cases though